x-pack/legacy/plugins/transform/index.ts

```diff
@@ -6,9 +6,10 @@
 
 import { Legacy } from 'kibana';
 import { resolve } from 'path';
+
+import { PluginInitializerContext } from 'src/core/server';
```

Comment:
I'm not seeing where this is used in this file 🤔

Reply:
Good find, VSCode didn't complain ... but the tests failed because of this :) Fixed in 22d2fb5.